### ISSUE TITLE
[3.5 beta] Crash when using skin placeholder.

### ISSUE THREAD

**Issue by pharan** [EsotericSoftware/spine-editor#104]
http://esotericsoftware.com/forum/Skin-Placeholder-problem-7064

https://youtu.be/I-UTkvQhUdM

From forum user "Moses"

There have a bug in new version of Spine Trial 3.5.00 beta(Old version no problem)
It will happen every time when I use this function
Thank you very much

1. Open the goblins example project
2. Select head (or any part of image)
3. New a Skin Placeholder to head slot
4. In a few second - Crashes

```
Spine Launcher 3.4.04 Trial
Mac OS X x86_64 10.11.6
Java 1.7.0_51 Esoteric Software
64-bit Server VM
Up to date: Spine Trial 3.5.00-beta
Spine 3.5.00-beta Trial, JGLFW
Intel Inc.
Intel HD Graphics 4000 OpenGL Engine
2.1 INTEL-10.14.73
Started.
Sorry, an unexpected error has occurred:
java.lang.NullPointerException
at com.esotericsoftware.spine.editor.Editor.a(SourceFile:654)
at com.esotericsoftware.spine.editor.Editor.b(SourceFile:822)
at ad.a(SourceFile:264)
at ad.e(SourceFile:229)
at mr.e(SourceFile:97)
at ai.run(SourceFile:191)
at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:251)
at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:733)
at java.awt.EventQueue.access$200(EventQueue.java:103)
at java.awt.EventQueue$3.run(EventQueue.java:694)
at java.awt.EventQueue$3.run(EventQueue.java:692)
at java.security.AccessController.doPrivileged(Native Method)
at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:76)
at java.awt.EventQueue.dispatchEvent(EventQueue.java:703)
at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:242)
at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:161)
at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:150)
at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:146)
at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:138)
at java.awt.EventDispatchThread.run(EventDispatchThread.java:91)
```

**Comment by pharan** [comment]
Forum OP said
"I buy Pro version already today
And It's no this proplem"

Maybe it had something to do with the Trial version? (based on the youtube video)

**Comment by NathanSweet** [comment]
Crash occurred when mousing over empty skin placeholders in the tree when a skin was visible. Fixed in 3.5.01-beta.

**Comment by pharan** [comment]
That's weird, 'cause that's exactly what I suspected. I tried to repro with goblins. Added a skin placeholder with nothing in it. It was fine.
